Makefile

```diff
@@ -3,7 +3,7 @@ ORG ?= kiagnose
 CHECKUP_IMAGE_NAME ?= kubevirt-dpdk-checkup
 CHECKUP_IMAGE_TAG ?= latest
 CHECKUP_GIT_TAG ?= $(shell git describe --always --abbrev=8 --tags)
-CHECKUP_BASE_IMAGE_TAG ?= latest
+CHECKUP_BASE_IMAGE_TAG ?= 9.2-717
```

Comment:
commit: `Makefile: Set a value to CHECKUP_BASE_IMAGE_TAG` - Can you elaborate? it's not clear for the description what is the problem and why it's worse on older versions.

Reply:
Each time you build the checkup image, the base image is the latest.
So when you build an old code, lets say from a year ago, it will have the latest base image, and not what was set a year ago.